contracts/Quest.sol

```diff
@@ -22,14 +22,13 @@ contract Quest is IQuest {
     // state variables
     bool public initialized;
     bool public started;
-    bool public extendedOnce;
-    bool public extendedTwice;
-    bool public extendedThrice;
     bool public beingDisputed;
     bool public finished;
     bool public rewarded;
     bool public withToken;
 
+    uint256 public MAX_EXTENSIONS;
```

Comment:
Immutable values can only be set inline or within the constructor, so we cant really use that here. But since there's actually no setter for the value, its somewhat immutable

Reply:
right, my bad (we don't use constructor)